### PR TITLE
New tutorials dl file

### PR DESCRIPTION
**Pull request by marscher**
## Changes

If "single_pass_weld.weldx" does not exist in tutorials_dir, it will be downloaded from IW repo.
The hash is being compared to the one observed from tagged version "weldx_0.5.0" of this repo.

## Checks

- [x] updated doc/

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -22,6 +22,9 @@
 import typing
 
 import traitlets
+from sphinx.util.logging import getLogger
+
+logger = getLogger("sphinx")
```
Comment (marscher) [inline doc/conf.py:27]: this should be "conf" or something

**Comment by CagtayFabry** [comment]
> @CagtayFabry Do you intend creating another tagged version any soon?

I will create a tag for weldx 0.5.0 👍

**Comment by marscher** [comment]
> > @CagtayFabry Do you intend creating another tagged version any soon?
> 
> I will create a tag for weldx 0.5.0 +1
> 
> [done]
Thank you for the quick response!

**Comment by marscher** [comment]
I hope we can ignore the deep source stuff for this file.

**Review by CagtayFabry — APPROVED** [review]
the asdf pytest errors should be gone once this is all rebased on master

**Comment by marscher** [comment]
is there a circular import in pint?!
`
  File "/home/runner/micromamba/envs/rtd/lib/python3.9/site-packages/pint/quantity.py", line 40, in <module>
    from ._typing import S, Shape, UnitLike, _MagnitudeType
ImportError: cannot import name 'S' from partially initialized module 'pint._typing' (most likely due to a circular import) (/home/runner/micromamba/envs/rtd/lib/python3.9/site-packages/pint/_typing.py)
`

**Comment by vhirtham** [comment]
> is there a circular import in pint?! `File "/home/runner/micromamba/envs/rtd/lib/python3.9/site-packages/pint/quantity.py", line 40, in <module> from ._typing import S, Shape, UnitLike, _MagnitudeType ImportError: cannot import name 'S' from partially initialized module 'pint._typing' (most likely due to a circular import) (/home/runner/micromamba/envs/rtd/lib/python3.9/site-packages/pint/_typing.py)`

The tutorials branch is not up to date. The pint fix of @CagtayFabry is still missing. I will update the branch and you can also delete the `single_pass_weld.wx` from the branch

**Comment by vhirtham** [comment]
Can you just quickly change the file extensions in the tutorials from `wx` to `weldx`. The tests should pass then.

**Comment by marscher** [comment]
> Can you just quickly change the file extensions in the tutorials from `wx` to `weldx`. The tests should pass then.

I guess it would be easier to change the output file name in the download function ;) Will do.

**Comment by marscher** [comment]
The file is currently only downloaded within the documentation, not in the top-level tutorials dir. I can add the same code to some util.py within the tutorials, so it can be used there. But I'd like to avoid this duplication.
One suggested change would be to move the tutorials to doc, or have some hidden cell within the notebooks to execute the code. Then I would move the download function to an utility module residing in the tutorials dir.

**Comment by CagtayFabry** [comment]
I think it would be best to simply move the tutorials to `doc/` (regardless of the solution here)

**Comment by vhirtham** [comment]
> I think it would be best to simply move the tutorials to `doc/` (regardless of the solution here)

Yes, but we can do this on the `new_tutorials` branch once all are merged.

**Comment by CagtayFabry** [comment]
I think this is ready @marscher @vhirtham

**Review by vhirtham — APPROVED** [review]

**Comment by vhirtham** [comment]
> I think this is ready @marscher @vhirtham

Should be ready, yes

**Pull request merged**